src/scss/initial/_headings.scss

```diff
@@ -37,7 +37,7 @@ h3 {
       var(--heading-start),
       var(--heading-end)
     );
-    -webkit-background-clip: text;
+    background-clip: text;
     background-clip: text;
```

Comment:
This seems like a mistake? Did you update a linter?

Reply:
Hrm, yeah -- this was changed by Stylelint: https://stylelint.io/user-guide/rules/property-no-vendor-prefix/